### PR TITLE
Update mimir distributed release 4.4.1-rc.0

### PR DESCRIPTION
**Pull request by lamida**
#### Which issue(s) this PR fixes or relates to

Fixes #<issue number>

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: operations/helm/charts/mimir-distributed/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-version: 4.4.0-weekly.236
-appVersion: r236
+version: 4.4.0
```
Comment (krajorama) [inline operations/helm/charts/mimir-distributed/Chart.yaml:2]:
```suggestion
version: 4.4.0-rc.0
```

Comment (lamida) [reply operations/helm/charts/mimir-distributed/Chart.yaml]: Updated in 4bc6cbd9f243b7f39e5bfcca5523f47f295fe28b

File: operations/helm/charts/mimir-distributed/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Grafana Mimir Helm chart
 
-Helm chart for deploying [Grafana Mimir](https://grafana.com/docs/mimir/v2.7.x/) or optionally [Grafana Enterprise Metrics](https://grafana.com/docs/enterprise-metrics/v2.6.x/) to Kubernetes. Derived from [Grafana Enterprise Metrics Helm chart](https://github.com/grafana/helm-charts/blob/main/charts/enterprise-metrics/README.md)
+Helm chart for deploying [Grafana Mimir](https://grafana.com/docs/mimir/v2.8.x/) or optionally [Grafana Enterprise Metrics](https://grafana.com/docs/enterprise-metrics/v2.6.x/) to Kubernetes. Derived from [Grafana Enterprise Metrics Helm chart](https://github.com/grafana/helm-charts/blob/main/charts/enterprise-metrics/README.md)
```
Comment (krajorama) [inline operations/helm/charts/mimir-distributed/README.md:3]:
```suggestion
Helm chart for deploying [Grafana Mimir](https://grafana.com/docs/mimir/v2.8.x/) or optionally [Grafana Enterprise Metrics](https://grafana.com/docs/enterprise-metrics/v2.8.x/) to Kubernetes. Derived from [Grafana Enterprise Metrics Helm chart](https://github.com/grafana/helm-charts/blob/main/charts/enterprise-metrics/README.md)
```

File: operations/helm/charts/mimir-distributed/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 
 See the [Grafana Mimir version 2.6 release notes](https://grafana.com/docs/mimir/v2.6.x/release-notes/v2.6/).
```
Comment (krajorama) [inline operations/helm/charts/mimir-distributed/README.md:5]: let's update this as well

File: operations/helm/charts/mimir-distributed/README.md.gotmpl [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 Helm chart for deploying [Grafana Mimir]({{ template "chart.homepage" . }}) or optionally [Grafana Enterprise Metrics](https://grafana.com/docs/enterprise-metrics/v2.6.x/) to Kubernetes. Derived from [Grafana Enterprise Metrics Helm chart](https://github.com/grafana/helm-charts/blob/main/charts/enterprise-metrics/README.md)
 
-See the [Grafana Mimir version 2.6 release notes](https://grafana.com/docs/mimir/v2.6.x/release-notes/v2.6/).
+See the [Grafana Mimir version 2.7 release notes](https://grafana.com/docs/mimir/v2.7.x/release-notes/v2.7/).
```
Comment (krajorama) [inline operations/helm/charts/mimir-distributed/README.md.gotmpl:5]:
```suggestion
See the [Grafana Mimir version 2.8 release notes](https://grafana.com/docs/mimir/v2.8.x/release-notes/v2.8/).
```

File: operations/helm/charts/mimir-distributed/README.md (hunk 2 above)
Comment (lamida) [reply operations/helm/charts/mimir-distributed/README.md:3]: Updated the template in 2581be11d011f0f5abddb3c0f000a6d903665711

File: operations/helm/charts/mimir-distributed/README.md (hunk 3 above)
Comment (lamida) [reply operations/helm/charts/mimir-distributed/README.md]: Updated the template in 2581be11d011f0f5abddb3c0f000a6d903665711

File: operations/helm/charts/mimir-distributed/README.md.gotmpl [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 Helm chart for deploying [Grafana Mimir]({{ template "chart.homepage" . }}) or optionally [Grafana Enterprise Metrics](https://grafana.com/docs/enterprise-metrics/v2.6.x/) to Kubernetes. Derived from [Grafana Enterprise Metrics Helm chart](https://github.com/grafana/helm-charts/blob/main/charts/enterprise-metrics/README.md)
```
Comment (krajorama) [inline operations/helm/charts/mimir-distributed/README.md.gotmpl:3]:
```suggestion
Helm chart for deploying [Grafana Mimir]({{ template "chart.homepage" . }}) or optionally [Grafana Enterprise Metrics](https://grafana.com/docs/enterprise-metrics/v2.8.x/) to Kubernetes. Derived from [Grafana Enterprise Metrics Helm chart](https://github.com/grafana/helm-charts/blob/main/charts/enterprise-metrics/README.md)
```

File: operations/helm/charts/mimir-distributed/README.md (hunk 2 above)
Comment (lamida) [reply operations/helm/charts/mimir-distributed/README.md:3]: Not updated actually. I will commit the suggestion instead.

**Review by krajorama — APPROVED** [review]

**Pull request merged**